packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -96,29 +101,19 @@ describe('AddresslistVoting', function () {
     );
   });
 
-  function addresslist(length: number): string[] {
-    let addresses: string[] = [];
-
-    for (let i = 0; i < length; i++) {
-      const addr = signers[i].address;
-      addresses.push(addr);
-    }
-    return addresses;
-  }
-
   describe('initialize: ', async () => {
     it('reverts if trying to re-initialize', async () => {
-      await voting.initialize(dao.address, votingSettings, addresslist(0));
+      await voting.initialize(dao.address, votingSettings, await addresses(0));
```

Comment:
Wouldn't it be easier/faster to get the array once at the beginning and keep it stored locally?

Reply:
You mean in an `beforeEach` block? 
The array length often varies across tests. I would keep it as is

Reply:
Removed the `addresses` helper in 0d329872360849b5a4a5a8aa53d88388309e8b4c